### PR TITLE
Archimedean spiral

### PR DESCRIPTION
**Pull request by cicklow**
Add Archimedean spiral

### CODE REVIEW

**Comment by cicklow** [comment]
sin titulo

**Comment by MTK4355** [comment]
Beautiful!

**Comment by SscSPs** [comment]
btw, you forgot to wipe your name off of the pokemon gui

File: PokemonGo.RocketAPI.Console/GUI.cs [hunk 1]
```diff
@@ -1499,5 +1505,15 @@ private void checkBox3_CheckedChanged(object sender, EventArgs e)
         {
 
         }
+
+        private void button4_Click(object sender, EventArgs e)
+        {
+            MessageBox.Show("Walk spiral getting Pokemon and only pokestops who are within 30 meters of the course.");
+        }
```
Comment (SscSPs) [inline PokemonGo.RocketAPI.Console/GUI.cs]: This will capture pokemons, and will use pokestops which are within 30 meters of the path projected.

**Comment by cicklow** [comment]
@SscSPs jajaj thanks XD

**Comment by toxic189090** [comment]
i almost feel like a spiral would look more like a bot

**Comment by SscSPs** [comment]
btw, the path taken is not sent to niantic.

**Pull request merged**